src/rpc/governance.cpp

```diff
@@ -226,6 +230,55 @@ UniValue gobject_prepare(const JSONRPCRequest& request)
 
     return tx->GetHash().ToString();
 }
+
+void gobject_list_prepared_help(CWallet* const pwallet)
+{
+    throw std::runtime_error(
+                "gobject list-prepared <limit>\n"
+                "Returns a list of governance objects prepared by this wallet with \"gobject prepare\" sorted by their creation time.\n"
+                + HelpRequiringPassphrase(pwallet) + "\n"
+                "\nArguments:\n"
+                "1. limit (numeric, optional) Maximal number of objects to return.\n"
+                );
+}
+
+UniValue gobject_list_prepared(const JSONRPCRequest& request)
+{
+    CWallet* const pwallet = GetWalletForJSONRPCRequest(request);
+    if (request.fHelp || (request.params.size() > 2)) {
+        gobject_prepare_help(pwallet);
+    }
+
+    if (!EnsureWalletIsAvailable(pwallet, request.fHelp))
+        return NullUniValue;
+
+    EnsureWalletIsUnlocked(pwallet);
+
+    int64_t nLimit = request.params.size() > 1 ? ParseInt64V(request.params[1], "limit") : std::numeric_limits<int64_t>::max();
+    if (nLimit <= 0) {
+        throw JSONRPCError(RPC_INVALID_PARAMETER, "limit needs to be greater 0");
```

Comment:
nit: a more common error message would be
```suggestion
        throw JSONRPCError(RPC_INVALID_PARAMETER, "Negative count");
```

Reply:
Hmm buuut in this case it would also complain with "Negative count" if you provide a `0` which would be false statement then? 🙈

Reply:
Just make it `if (nLimit < 0) {` then :D

Reply:
Allowing to query 0 objects feels a bit strange just because there is no point in doing this :D but ok, i can do that to align it with the others.. 😅

Reply:
See 7befd65